tests/common.go

```diff
@@ -5043,21 +5042,29 @@ func StartTorpedoTest(testName, testDescription string, tags map[string]string,
 // enableAutoFSTrim on supported PX version.
 func EnableAutoFSTrim() {
 	nodes := node.GetWorkerNodes()
-	pxVersion, err := Inst().V.GetPxVersionOnNode(nodes[0])
-	log.FailOnError(err, "Is autofstrim supported on the cluster ?")
-	log.Infof("PX version %s", pxVersion)
-	pxVersionList := []string{}
-	pxVersionList = strings.Split(pxVersion, ".")
-	majorVer, err := strconv.Atoi(pxVersionList[0])
-	minorVer, err := strconv.Atoi(pxVersionList[1])
-	if majorVer < 2 || (majorVer == 2 && minorVer < 10) {
-		log.Warnf("Auto FSTrim cannot be enabled on PX version %s", pxVersion)
-	} else {
-		err = Inst().V.SetClusterOpts(nodes[0], map[string]string{
-			"--auto-fstrim": "on"})
-		log.FailOnError(err, "Autofstrim is enabled on the cluster ?")
-		log.Infof("Auto FSTrim enabled on the cluster")
+	var isPXNodeAvailable bool
+	for _, n := range nodes {
+		if n.IsStorageDriverInstalled {
+			isPXNodeAvailable = true
+			pxVersion, err := Inst().V.GetPxVersionOnNode(nodes[0])
+			log.FailOnError(err, "Is autofstrim supported on the cluster ?")
+			log.Infof("PX version %s", pxVersion)
+			pxVersionList := []string{}
+			pxVersionList = strings.Split(pxVersion, ".")
+			majorVer, err := strconv.Atoi(pxVersionList[0])
+			minorVer, err := strconv.Atoi(pxVersionList[1])
+			if majorVer < 2 || (majorVer == 2 && minorVer < 10) {
+				log.Warnf("Auto FSTrim cannot be enabled on PX version %s", pxVersion)
+			} else {
+				err = Inst().V.SetClusterOpts(nodes[0], map[string]string{
+					"--auto-fstrim": "on"})
+				log.FailOnError(err, "Autofstrim is enabled on the cluster ?")
+				log.Infof("Auto FSTrim enabled on the cluster")
+			}
+			break
+		}
 	}
+	dash.VerifyFatal(isPXNodeAvailable, true, "No PX node available in the cluster")
```

Comment:
I don't think you need to introduce new var `isPXNodeAvailable`, I think just using `n.IsStorageDriverInstalled` is enough

Reply:
Calling Inst().V.IsPxInstalled(pxNode) instead of IsStorageDriverInstalled
Keeping isPXNodeAvailable to assert when no PX node is available.